src/tasks/AotCompilerTask/MonoAOTCompiler.props

```diff
@@ -15,7 +15,15 @@
     <MonoAOTCompilerDefaultProcessArguments Include="-O=gsharedvt" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetOS)' == 'Android'">
+    <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'arm'" Include="mtriple=armv7-linux-gnueabi-thumbv7s" />
```

Comment:
I guess we should pass the triple for the other architectures too?

Reply:
We've only had the problem on arm thus far.